app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/vm.rb

```diff
@@ -38,6 +41,27 @@ def raw_destroy
     end
   end
 
+  def self.params_for_create_snapshot
```

Comment:
The `params_for_create_snapshot` are expected to be an instance method not a class method.  This likely would cause https://github.com/ManageIQ/manageiq-api/blob/master/app/controllers/api/subcollections/snapshots.rb#L51 to raise an exception